client/sites/staging-site/components/staging-site-card/card-content/staging-site-creation-card/staging-site-creation-card-content.scss

```
@@ -0,0 +1,69 @@
+@import "@wordpress/base-styles/mixins";
+
+.staging-site-creation-card {
+	padding: 2rem;
+	display: flex;
+	justify-content: space-between;
+	width: 100%;
+	max-width: 100%;
+	box-sizing: border-box;
+	gap: 1rem;
+	position: relative;
+	overflow: hidden;
+	background-image: url(./site-creation-graphic.svg);
+	background-repeat: no-repeat;
+	background-position: center right;
+	background-size: auto 100%;
+
+	@media screen and (max-width: 768px) {
```

I think we could consider adjusting formatting throughout this SCSS file, e.g. adding spaces inside parenthesis:

```suggestion
	@media screen and ( max-width: 768px ) {
```

---

Sounds good, adjusted in https://github.com/Automattic/wp-calypso/pull/104520/commits/cc26a46121b5b9c081ad0bbac1769cbefcea358c